rnaseq-cgl-pipeline/wrapper.py

```diff
@@ -0,0 +1,101 @@
+import argparse
+import os
+import shutil
+import subprocess
+import json
+import logging
+from uuid import uuid4
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger()
+
+
+def call_pipeline(mount, args):
+    uuid = 'Toil-RNAseq-' + str(uuid4())
+    if not os.path.exists(mount + uuid):
+        os.makedirs(os.path.join(mount, uuid))
+    os.environ['PYTHONPATH'] = '/opt/rnaseq-pipeline/src'
+    command = ['python', '-m', 'toil_scripts.rnaseq_cgl.rnaseq_cgl_pipeline',
+               os.path.join(mount, 'jobStore'),
+               '--retryCount', '1',
+               '--output-dir', mount,
+               '--workDir', os.path.join(mount, uuid),
+               '--star-index', 'file://'+ args.star,
+               '--rsem-ref', 'file://' + args.rsem,
+               '--kallisto-index', 'file://' + args.kallisto,
+               '--sample-urls']
+    command.extend(['file://' + x for x in args.samples])
```

Comment:
You can leave of the square brackets. That will make it more efficient (making it a generator expression as opposed to a list comprehension) and more concise.